### PR TITLE
Remove neutron lbaas v1 version code

### LINKED ISSUE #1833: remove lbaas code ? ?
**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line: 
>
> /kind bug
> /kind feature

**What happened**:

OCCM we claimed lbaas is deprecated since this commit `commit 994df0e7069c54a169e7fbf71c72898ca3cae696` 
it's about 2 years ago and should we consider remove all those code for neutron lbaas and 
also make octavia default (so no use-octavia flag)

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
- openstack-cloud-controller-manager(or other related binary) version:
- OpenStack version:
- Others:

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

**Which issue this PR fixes(if applicable)**:
fixes #1833

**Special notes for reviewers**:

**Release note**:

```release-note
neutron lbaas is not supported anymore, we should not keep those data, removed all neutron lbaas v1 code along with the UseOctavia flag, action required.
```

### CODE REVIEW

**Comment by chrigl** [comment]
Just one thing. I think we should update the main README.md, and add the removal to Note

**Comment by jichenjc** [comment]
>>>Just one thing. I think we should update the main README.md, and add the removal to Note

good point, updated~

**Comment by ramineni** [comment]
> @ramineni: the `/release-note` and `/release-note-action-required` commands have been deprecated. Please edit the `release-note` block in the PR body text to include the release note. If the release note requires additional action include the string `action required` in the release note. For example:
> 
> ```
> ```release-note
> Some release note with action required.
> ```
> ```

@jichenjc could you add action required in release note, so that easier to check before upgrade

**Comment by jichenjc** [comment]
@ramineni I think I already have the release note?

**Comment by jichenjc** [comment]
@ramineni added `action required.` into release notes, thanks

**Comment by ramineni** [comment]
@jichenjc Thanks. And also please remove the flag `use-octavia` from the test manifests and documents.

**Comment by jichenjc** [comment]
@ramineni updated, thanks~

**Comment by jichenjc** [comment]
@chrigl @zetaab can you help review this?

**Comment by jichenjc** [comment]
@chrigl @zetaab can you help review this ? Thanks

**Comment by jichenjc** [comment]
@shaardie @chrigl can you help check this?

File: README.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ This Repository hosts various plugins relevant to OpenStack and Kubernetes Integ
 
 * Cinder Standalone Provisioner, Manila Provisioner and Cinder FlexVolume Driver were removed since release v1.18.0.
 * Version 1.17 was the last release of Manila Provisioner, which is unmaintained from now on. Due to dependency issues, we removed the code from master but it is still accessible in the [release-1.17](https://github.com/kubernetes/cloud-provider-openstack/tree/release-1.17) branch. Please consider migrating to Manila CSI Plugin.
+* Start from release v1.25.0, neutron lbaasv1 support is removed and only Octavia is supported.
```
Comment (shaardie) [inline README.md:20]: This patch did not make in the 1.25 release. So probably the Readme update should state the next release?

**Review by shaardie — COMMENTED** [review]
This branch does have conflicts as far as I can see, so it can probably not being merged in this form.

It's too bad, because removing all this old code would greatly improve the readability and code quality.

**Comment by jichenjc** [comment]
@chrigl can you check and merge this ? last for long time and introduce unnecessary code update for some time

**Comment by jichenjc** [comment]
/approve

this has been long time and I think no one will approve it 
with one lgtm, I am going to approve it

**Pull request merged**